client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -122,6 +122,21 @@ struct SubscribeRequest {
   std::variant<EventGroupRequest, LegacyEventRequest> request;
 };
 
+struct GetRecentSnapshotRequest {
```

Comment:
Logically, maybe this data structure should be called a response. We have the same piece of data in the `GetRecentSnapshotResponse` protobuf message. Please correct me if I am misunderstanding the intention.

The request itself doesn't have any data members, as of now.